From 4d30822402665c35e7bc76ed7c94bbbefa801e36 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Sat, 20 Feb 2021 19:10:58 +0100
Subject: [PATCH 1/1] Solus tweaks to default kickoff

- Add shortcuts for Software Center in menu
- Add Solus icon to kickoff menu
---
 applets/kickoff/package/contents/config/main.xml | 7 ++++---
 applets/kickoff/package/metadata.desktop         | 2 +-
 2 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index f44001f..16bb53f 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -11,11 +11,12 @@
         </entry>
         <entry name="icon" type="String">
             <label>The name of the icon used in the compact representation (e.g. on a small panel).</label>
-            <default>start-here-kde</default>
+            <default>start-here-solus</default>
         </entry>
         <entry name="favorites" type="StringList">
             <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-            <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover.desktop</default>
+            <default>preferred://browser,systemsettings.desktop,org.kde.dolphin.desktop,org.kde.kate.desktop
+,solus-sc.desktop</default>
         </entry>
         <entry name="systemFavorites" type="StringList">
             <label>List of system action favorites.</label>
@@ -31,7 +32,7 @@
         </entry>
         <entry name="systemApplications" type="StringList">
             <label>List of applications at the top of the "Computer" tab.</label>
-            <default>systemsettings.desktop,org.kde.kinfocenter.desktop,org.kde.discover.desktop</default>
+            <default>systemsettings.desktop,org.kde.kinfocenter.desktop,solus-sc.desktop</default>
         </entry>
         <entry name="favoritesDisplay" type="Int">
             <label>How to display favorites: 0 = Grid, 1 = List</label>
diff --git a/applets/kickoff/package/metadata.desktop b/applets/kickoff/package/metadata.desktop
index ddaee52..e703521 100644
--- a/applets/kickoff/package/metadata.desktop
+++ b/applets/kickoff/package/metadata.desktop
@@ -166,7 +166,7 @@ Comment[wa]=Enondeu po lancî des programes
 Comment[x-test]=xxLauncher to start applicationsxx
 Comment[zh_CN]=应用程序的启动器
 Comment[zh_TW]=應用程式啟動器
-Icon=start-here-kde
+Icon=start-here-solus
 Type=Service
 X-KDE-ServiceTypes=Plasma/Applet
 
-- 
2.30.0

